skops/io/tests/test_audit.py

```diff
@@ -146,6 +148,15 @@ def __init__(self):
     assert not hasattr(temp, "b")
 
 
+@pytest.mark.parametrize(
+    "estimator", get_tested_estimators(), ids=_get_check_estimator_ids
+)
+def test_sklearn_trusted_types(estimator):
+    untrusted_types = get_untrusted_types(data=dumps(estimator))
+    sklearn_untrusted_types = [t for t in untrusted_types if t.startswith("skelarn.")]
+    assert len(sklearn_untrusted_types) == 0
```

Comment:
This is correct as what we need to test, but it's adding a round of dumping estimators to the whole test suite. I wonder if we should add it right after where we do `get_untrusted_types` in the existing tests, instead of adding a new round of tests.

WDYT @skops-dev/maintainers

Reply:
I would be fine either way, having different tests for different things makes sense but performance also matters.

Btw., there is a typo, `"skelarn."`

Reply:
(Oops, fixed the typo in the meantime)

Reply:
I moved this `assert` into `test_can_persist_fitted`. Some of the `sklearn` types that are still untrusted were revealed by testing this with a fitted estimator. (see comment below)